internal/common/gamestate/gamestate.go

```diff
@@ -27,7 +27,7 @@ type GameState struct {
 	ForagingHistory map[shared.ForageType][]foraging.ForagingReport
 
 	// IIGO History
-	IIGOHistory []shared.Accountability
+	IIGOHistory map[uint][]shared.Accountability
```

Comment:
add a comment above saying it is indexed by turn?

Reply:
On it